SingularityUI/app/actions/api/base.es6

```diff
@@ -23,12 +23,12 @@ export function buildApiAction(actionName, opts = {}, keyFunc = undefined) {
     return { type: CLEAR };
   }
 
-  function started(key = undefined) {
-    return { type: STARTED, key };
+  function started(key = undefined, options = undefined) {
```

Comment:
That is a good point. I think I was copying the style of the key argument, but when I think about it it really doesn't make much sense. I'll take that out.